sklearn/feature_extraction/tests/test_image.py

```diff
@@ -170,6 +171,26 @@ def test_extract_patches_max_patches():
                   max_patches=-1.0)
 
 
+def test_extract_patch_same_size_image():
+    face = downsampled_face
+    i_h, i_w = face.shape
+
+    # Request patches of the same size as image
+    # Should return just the single patch a.k.a. the image
```

Comment:
I would like probably to assert the number of returned patch

Reply:
Understood